value/src/test/java/com/google/auto/value/processor/PropertyAnnotationsTest.java

```diff
@@ -152,26 +153,37 @@ private JavaFileObject expectedCode(List<String> annotations) {
     return JavaFileObjects.forSourceLines("foo.bar.AutoValue_Baz", lines);
   }
 
+  
   private void assertGeneratedMatches(
       List<String> imports,
       List<String> annotations,
-      List<String> expectedAnnotations) {
+      List<String> expectedAnnotations, String expectedConstructorParamAnnotation) {
 
     JavaFileObject javaFileObject = sourceCode(imports, annotations);
-    JavaFileObject expectedOutput = expectedCode(expectedAnnotations);
+    JavaFileObject expectedOutput = expectedCode(expectedAnnotations, expectedConstructorParamAnnotation);
 
     assert_().about(javaSource())
         .that(javaFileObject)
         .processedWith(new AutoValueProcessor())
         .compilesWithoutError()
         .and().generatesSources(expectedOutput);
+
+  }
+  
+  private void assertGeneratedMatches(
+      List<String> imports,
+      List<String> annotations,
+      List<String> expectedAnnotations) {
+    assertGeneratedMatches(imports, annotations, expectedAnnotations, null);
   }
 
   public void testSimpleAnnotation() {
     assertGeneratedMatches(
         ImmutableList.of("import javax.annotation.Nullable;"),
         ImmutableList.of("@Nullable"),
-        ImmutableList.of("@javax.annotation.Nullable"));
+        ImmutableList.of("@javax.annotation.Nullable"),
+        "@javax.annotation.Nullable"
+        );
```

Comment:
Please join this to the end of the previous line. (I find that the assertGeneratedMatches method is now quite cryptic but that's a matter for another day.)